ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -3501,11 +3501,11 @@ signal_list_t* create_operation_node(ast_node_t* op, signal_list_t** input_lists
                 input_lists[1]->pins[j] = NULL;
             }
             /* allocate the pins needed */
-            allocate_more_input_pins(operation_node, log2(op->children[1]->types.vnumber->get_value()));
+            allocate_more_input_pins(operation_node, op->children[1]->types.vnumber->size());
```

Comment:
you cant get the size either!! is it a NUMBER ? if not,  types.vnumber is null, or as good as null. dont touch it